admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -148,16 +175,22 @@ export const ApplicationPage: React.FunctionComponent = () => {
                                                 helperText='This is the name you want to display on the UI'>
                                                 <TextInput
                                                     type='text'
-                                                    value={ eventType.displayName }
+                                                    defaultValue={ eventType.displayName }
```

Comment:
I don't think we need to change this to defaultValue. value seems fine.

```suggestion
                                                    value={ eventType.displayName }
```